### DISCUSSION TITLE
Открыть файл на локальном компьютере с помощью ассоциированного приложения.

### DISCUSSION THREAD

**Discussion by vldekov** [totumonline/totum-ru-issues-and-discussions#395] [Как это сделать?] [answered]
Есть задача открыть файл на локальном компьютере с помощью ассоциированного приложения. Например, EXCEL файл.
Помогите написать код действия.

**Comment by erpic-ru** [comment]
[linktoscript](https://ru.docs.totum.online/functions#fn-linkToScript) + [что-то такое](https://habr.com/ru/articles/708760/)

**Reply by vldekov** [reply]
= :linkToScript(uri: "C:\ClientSOCC\КомплектТД.txt") - не работает

**Reply by LostAlly** [reply]
«Локальные» url начинаются с file:///
Вроде бэкслеши нужно заменить на слеши.

**Reply by ttm-alexeypolunin** [reply]
Зависит от поддержки в ОС. Вот так вызывается локальный Telegram:

```
<a href="tg://resolve?domain=username">Открыть Telegram</a>
```

По конкретному приложению спросите у GPT

**Reply by ttm-alexeypolunin** [reply, marked as answer]
История в том, что файл находится на сервере, а ваше локальное приложение на локальном компе. Для того, чтобы эксельки открывались из тотум будет интеграция с Only Office в Ultra-версии. Возможно когда-нибудь будет локальная версия Тотум, которая будет копировать файл с сервера в tmp-папку, вызывать локальное приложение и потом возвращать на сервер. Но я бы не ждал, тк это отдельная разработка, которая тянет миллионов на 10-15, а у меня их нет.
